tidb-resource-control.md

```diff
@@ -43,21 +43,48 @@ Currently, the resource control feature has the following limitations:
 
 ## What is Request Unit (RU)
 
-Request Unit (RU) is a unified abstraction unit in TiDB for system resources, which currently includes CPU, IOPS, and IO bandwidth metrics. The consumption of these three metrics is represented by RU according to a certain ratio.
+Request Unit (RU) is a unified abstraction unit in TiDB for system resources, which currently includes CPU, IOPS, and IO bandwidth metrics. It is used to indicate the amount of resources consumed by a single request to the database. The number of RUs consumed by a request depends on a variety of factors, such as the type of operation, and the amount of data being retrieved or modified. Currently, the RU contains statistics for the resources in the following table:
```

Comment:
```suggestion
Request Unit (RU) is a unified abstraction unit in TiDB for system resources, which currently includes CPU, IOPS, and IO bandwidth metrics. It is used to indicate the amount of resources consumed by a single request to the database. The number of RUs consumed by a request depends on a variety of factors, such as the type of operations, and the amount of data being queried or modified. Currently, the RU contains consumption statistics for the resources in the following table:
```

Reply:
reason: using "consumption statistics" to differentiate it from https://docs.pingcap.com/tidb/dev/statistics